cmd/osm-controller/validate.go

```diff
@@ -24,10 +24,6 @@ func validateCLIParams() error {
 		return errors.Errorf("Please specify the webhook configuration name using --webhook-config-name")
 	}
 
-	if caBundleSecretName == "" {
-		return errors.Errorf("Please specify the CA bundle secret name using --ca-bundle-secret-name")
-	}
-
 	if caBundleSecretName == "" {
 		return errors.Errorf("Please specify the CA bundle secret name using --ca-bundle-secret-name containing the cert-manager CA at 'ca.crt'")
```

Comment:
Is this a bit verbose? Not sure if the run-on sentence here makes sense

Reply:
I'm not totally sure what it means tbh, just kept the longer one because I figured it must've had some significance when it was added
cc: @shashankram ?

Reply:
I commented on this here: https://github.com/openservicemesh/osm/pull/3462#discussion_r640987740